src/core/plugins/json-schema-2020-12-samples/fn/class/OptionRegistry.js

```diff
@@ -4,7 +4,17 @@
 import Registry from "./Registry"
 
 class OptionRegistry extends Registry {
-  #defaults = {}
+  #defaults = {
+    randomEnabled: false,
+    random: Math.random,
+    minInt: 0,
+    maxInt: 2147483647,
+    minLen: 4,
+    maxLen: 10,
+    minDateTime: new Date("1970-01-01T00:00:00.000Z"),
+    maxDateTime: new Date("2038-01-19T00:00:00.000Z"),
+    maxRandExp: 100,
```

Comment:
```suggestion
    defaultRandExpMax: 10,
```

Reply:
Yeah 10 is fine which is what I initially thought of, but note that it will change the default behavior of randExp which is 100. But I think it's for the better to keep samples shorter and more readable.